eo-maven-plugin/src/main/java/org/eolang/maven/util/Walk.java

```diff
@@ -119,16 +119,23 @@ private static Stream<String> stream(final Collection<String> globs) {
      * @return List
      * @throws IOException If fails
```

Comment:
@volodya-lombrozo doesn't throw any more

Reply:
@mximp Indeed. Removed. Thank you.